tests/unit/s2n_random_test.c

```diff
@@ -432,12 +569,13 @@ static int s2n_random_test_case_failure_cb(struct random_test_case *test_case)
     return EXIT_SUCCESS;
 }
 
-#define NUMBER_OF_RANDOM_TEST_CASES 3
+#define NUMBER_OF_RANDOM_TEST_CASES 4
 struct random_test_case random_test_cases[NUMBER_OF_RANDOM_TEST_CASES] = {
```

Comment:
Nit, but you can avoid needing to keep these in sync by not specifying a length for random_test_cases and calling s2n_array_len(random_test_cases) whenever you need the length.

You couldn't call s2n_array_len to set the size of proc_ids, but looking at the code, why do you need an array of proc_ids? It looks like you only use one at a time. If I'm missing something and you do need the array, you can always just manually call (sizeof(random_test_cases) / sizeof(random_test_cases[0])) or make a little macro for it.

```suggestion
struct random_test_case random_test_cases[] = {
```